client/package.json

```diff
@@ -31,5 +31,6 @@
     "not dead",
     "not ie <= 11",
     "not op_mini all"
-  ]
+  ],
+  "proxy": "http://localhost:3001"
```

Comment:
I've got error: "Can't resolve axios and jwt-decode". Do we need to add them to project dependencies?

Reply:
Yep! Please run npm I axios and jwt-decode! I lost them after merging from dev(((
I'll added them before merging...